source/developers/in-context-editing-ftl.rst

```diff
@@ -0,0 +1,368 @@
+|| path          || Yes                         || the path to the component. Typically this is   |
+||               ||                             || simply contentModel.storeUrl                   |
++----------------+------------------------------+-------------------------------------------------+
+|| ice           || No                          || true or false. If true the component will      |
+||               ||                             || automatically render ICE (in context editing)  |
+||               ||                             || controls for you. This is helpful on simple    |
+||               ||                             || components. Larger components may be so complex|
+||               ||                             || that multiple ice elements make sense. In the  |
+||               ||                             || latter case omit this attribute or set it to   |
+||               ||                             || false and manually add your own ICE attributes |
+||               ||                             || to the component template                      |
++----------------+------------------------------+-------------------------------------------------+
+|| iceGroup      || No (unless path is not      || the label/id assigned to iceGroup on           |
+||               || supplied)                   || fields in your content model.                  |
++----------------+------------------------------+-------------------------------------------------+
+|| component     || No                          || a |SiteItem| object                            |
++----------------+------------------------------+-------------------------------------------------+
+
+Example
+
+    .. code-block:: guess
+
+	    <img <@studio.componentAttr path=contentModel.storeUrl ice=true /> src="${contentModel.image!""}" alt="${contentModel.alttext!""}" />
+
+|
+
+.. note:: Remember to have an item selector control in the form definition for each drop zone
+
+--------------
+Engine Support
+--------------
+
+At the top of your page or component (whatever it is you are rendering, include the following) import:
+
+    .. code-block:: guess
+
+	    <#import "/templates/system/common/crafter-support.ftl" as crafter/>
+
+|
+
+Components
+----------
+
+Render Component
+----------------
+
+Need to render a sub component of some kind? 
+
+    .. code-block:: guess
+
+	    <@renderComponent component=module />
+
+|
+
+Render Components
+-----------------
+
+Need to iterate through a list of components and render them WITHOUT any additional markup?
+
+
+    .. code-block:: guess
+
+	    <@crafter.renderComponents componentList=contentModel.bottomPromos />
+
+|
+
+Render RTE (Rich Text Editor Components)
+----------------------------------------
+
+Have components that are inserted in to the rich text editor and need to render them?
+
+    .. code-block:: guess
+
+	    <@crafter.renderRTEComponents />
```

Comment:
do we still support this in the new RTE?

Reply:
Maybe we should ask @Jose-Vega, is `<@crafter.renderRTEComponents />` still supported in the new RTE? Thanks!

Reply:
@jvega190 see above.

@alhambrav please note jvega's handle above.